### ISSUE TITLE
Index name for class java.lang.String name rel: false idx: true must differ from the default name (SDN 3.0) [DATAGRAPH-437]

### ISSUE THREAD

**Issue by spring-projects-issues** [spring-projects/spring-data-neo4j#1002]
**[Grégoire COLBERT](https://jira.spring.io/secure/ViewProfile.jspa?name=geceo)** opened **[DATAGRAPH-437](https://jira.spring.io/browse/DATAGRAPH-437?redirect=false)** and commented

After upgrading from SDN 2.3.4 to 3.0.0.RELEASE, I have two tests that fail with :

Index name for class java.lang.String name rel: false idx: true must differ from the default name: BesoinNode

Please see StackOverflow for the detailed problem.

To sum up, when upgrading from SDN 2.3.4 to SDN 3.0.0.RELEASE, the `@Indexed`(indexName = "indexOfNeeds") was replaced by `@Indexed`, so that the new index system, based on labels, was used.

But this leads to the aforementionned exception on the calls to IndexRepository#findByPropertyValue(String, Object)

---

**Reference URL:** http://stackoverflow.com/q/22111928/1528942
